src/Shared/FileUtilities.cs

```diff
@@ -1078,7 +1078,13 @@ internal static string MakeRelative(string basePath, string path)
             {
                 sb.Append(splitPath[i]).Append(Path.DirectorySeparatorChar);
             }
-            sb.Length--;
+
+            int length = fullPath.Length;
+            if (length < 1 || fullPath[length - 1] != Path.DirectorySeparatorChar)
```

Comment:
```suggestion
            if (fullPath[fullPath.Length - 1] != Path.DirectorySeparatorChar)
```

length is a little too generic to be useful, but this is a fairly common idiom, so I'd just simplify it like this.

Also, I don't think we need to check length < 1, do we? Can Path.GetFullPath return something of length 0?